.npmignore

```diff
@@ -1,3 +1,4 @@
 # Exclude everything but the contents of the dist directory.
 **/*
 !dist/**
+!src/css/**
```

Comment:
I don't think we do? Seems like that's a key part of the CSS, but I'll leave that up to Matt. I don't think it's hurting anything to include them.